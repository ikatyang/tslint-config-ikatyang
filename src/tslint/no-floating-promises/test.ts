// good

function good1() {
  return Promise.resolve(null);
}

// bad

function bad1() {
  Promise.resolve(null);
}
