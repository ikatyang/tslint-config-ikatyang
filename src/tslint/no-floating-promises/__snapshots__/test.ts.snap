// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

function good1() {
  return Promise.resolve(null);
}

// bad

function bad1() {
  Promise.resolve(null);
  ~~~~~~~~~~~~~~~~~~~~~  [Promises must be handled appropriately]
}

`;
